### ISSUE TITLE
ERROR TypeError: Cannot read property 'target' of undefined at ImageCropperComponent.loadImageOnChanges

### ISSUE THREAD

**Issue by vadim-lisovyi** [Mawi137/ngx-image-cropper#302]
Use sample from README
and get this error
```
ERROR TypeError: Cannot read property 'target' of undefined
    at ImageCropperComponent.loadImageOnChanges (ngx-image-cropper.js:333)
    at ImageCropperComponent.ngOnChanges (ngx-image-cropper.js:303)
    at checkAndUpdateDirectiveInline (core.js:33251)
    at checkAndUpdateNodeInline (core.js:46186)
    at checkAndUpdateNode (core.js:46125)
    at debugCheckAndUpdateNode (core.js:47148)
    at debugCheckDirectivesFn (core.js:47091)
    at Object.eval [as updateDirectives] (EditAvatarComponent.html:16)
    at Object.debugUpdateDirectives [as updateDirectives] (core.js:47079)
    at checkAndUpdateView (core.js:46090)
```

Angular CLI: 9.1.0
Node: 12.13.1
ngx-image-cropper: 3.1.4

**Comment by Mawi137** [comment]
This was fixed in 3.1.5. Can you upgrade and try again?

**Comment by vadim-lisovyi** [comment]
Yes indeed - the problem is resolved.
Thanks!
